data/postupgrade.d/copy_clean_conf.sh

```diff
@@ -19,6 +19,6 @@ do
         echo "Config file $NEW_FILE does not exists."
     fi
     echo "Copy file $file to $NEW_FILE"
-    cp $file $NEW_FILE
+    cp -ar $file $NEW_FILE
```

Comment:
You're right, the _-r_ is not needed. Each file from within the cleanconf dir tree is provided separately by the _find_ command.